chainer/functions/gaussian.py

```diff
@@ -0,0 +1,70 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Gaussian(function.Function):
+
+    """Gaussian sampling function.
+
+    It takes mean and log(variance) as input and output sample drawn from
```

Comment:
It is helpful to add a supplemental comment that this is the behavior of forward propagation.